src/SumoCoders/FrameworkCoreBundle/Resources/assets/js/Framework/Link.js

```diff
@@ -1,4 +1,7 @@
 export class Link {
+  constructor () {
+    this.initLink()
```

Comment:
Can't the initLink code not come in the constructor?

Reply:
Wouldn't that make the constructor a bit messy?